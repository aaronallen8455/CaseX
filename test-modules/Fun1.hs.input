module LambdaCase1 where

import qualified Import as I

main :: IO ()
main = pure ()

test :: I.Foo -> Bool
test SPLIT = True
