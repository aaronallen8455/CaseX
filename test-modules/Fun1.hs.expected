import qualified Import as I

main :: IO ()
main = pure ()

test :: I.Foo -> Bool
test I.Foo = True
test I.Bar = True
test I.Baz = True
